### ISSUE TITLE
Link to release notes whenever gauge plugin updates

### ISSUE THREAD

**Issue by zabil** [getgauge/gauge-vscode#30]
Plugins in vs code can upgrade automatically.
Whenever gauge plugin updates, show up an info bar, with the message.
"Gauge plugin updated to version 0.4.x" with a button "View release notes" and "Do not show this again"

Clicking "View release notes" takes the user to github releases page for that version.

**Comment by riju91** [comment]
It should be available in gauge-vscode nightly >= 2017-11-29

**Comment by sguptatw** [comment]
"Do not show this again" box should be available. On clicking this next upgrade should not show the message.

**Comment by zabil** [comment]
@sguptatw anything left in this?

**Comment by sguptatw** [comment]
Have done dev box on this. However this card can be tested only once VS-Code is released.

**Comment by sguptatw** [comment]
No longer relevant as this popup box will not be shown. The Welcome Page will be displayed instead.
